src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -156,5 +165,16 @@ export default class SimidMapCreative extends BaseSimidCreative {
       map: map,
       title: 'Current Position'
     });
+    let currentUserSession_ = this.newUserSession_;
+    map.addListener('zoom_changed', function() {
+      currentUserSession_.userInteractsWithMap();
+    });
+    map.addListener('click', function() {
+      currentUserSession_.userInteractsWithMap();
+    });
+    map.addListener('drag', function() {
+      currentUserSession_.userInteractsWithMap();
+    });
```

Comment:
IMO this seems a little repetitive. try something like this perhaps:
['click','zoom_changed', 'drag'].forEach( event => 
    map.addListener(event, () => this.newUserSession_.userInteractsWithMap())
);

Reply:
Good suggestion!
Some event listeners will take in an array of events, i.e. `map.addListener(['click', 'drag', 'zoom_changed'], () => {})`, so if the google maps api allows for that, that would be good too.

Reply:
done